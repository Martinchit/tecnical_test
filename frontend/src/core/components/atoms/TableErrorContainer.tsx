import styled from 'styled-components';

export const TableErrorContainer = styled.td`
  border-top: none !important;
  padding-top: 0 !important;
`;
