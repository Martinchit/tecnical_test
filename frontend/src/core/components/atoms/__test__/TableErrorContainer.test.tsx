import React from 'react';
import { shallow } from 'enzyme';

import { TableErrorContainer } from '../TableErrorContainer';

describe('TableErrorContainer', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<TableErrorContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
