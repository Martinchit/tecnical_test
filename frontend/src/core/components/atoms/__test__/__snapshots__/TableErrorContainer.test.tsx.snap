// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableErrorContainer renders correctly 1`] = `
<td
  className="sc-AxjAm cFFbws"
/>
`;
